bootstrap.js

```diff
@@ -1973,15 +2024,7 @@ var windowListener = {
 							break;
 						case 'extensions.tabtree.position':
 							let firstVisibleRow = tree.treeBoxObject.getFirstVisibleRow();
-							if (Services.prefs.getIntPref('extensions.tabtree.position') === 1) {
-								browser.appendChild(fullscrToggler);
-								browser.appendChild(splitter);
-								browser.appendChild(sidebar);
-							} else {
-								browser.insertBefore(fullscrToggler, appcontent);
-								browser.insertBefore(sidebar, appcontent);
-								browser.insertBefore(splitter, appcontent);
-							}
+							setTTPos(Services.prefs.getIntPref('extensions.tabtree.position'));
 							tree.view = view;
```

Comment:
And therefore `tree.treeBoxObject` isn't initialized properly (a <tree> element is very buggy as it turned out). I think it could be worked around with a listener on the sidebar showing/hiding (and several conditions throughout `bootstrap.js` where treeBoxObject and tree.view are used in order not to use them when the bookmarks sidebar is hidden). I can try to do it.
Anyway, thanks a lot. Honestly I had no idea how to work with the built-in sidebar.